openstack/networking/v2/extensions/layer3/portforwarding/requests.go

```diff
@@ -1,6 +1,53 @@
 package portforwarding
 
-import "github.com/gophercloud/gophercloud"
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+type ListOptsBuilder interface {
+	ToPortForwardingListQuery() (string, error)
+}
+
+// ListOpts allows the filtering and sorting of paginated collections through
+// the API. Filtering is achieved by passing in struct field values that map to
+// the port forwarding attributes you want to see returned. SortKey allows you to
+// sort by a particular network attribute. SortDir sets the direction, and is
+// either `asc' or `desc'. Marker and Limit are used for pagination.
+type ListOpts struct {
```

Comment:
We need to add `limit` (int) and `marker` (string) here.

Gophercloud's paging automatically sets limit and marker when paging internally, but if the user wanted to limit the results themselves, we need that in `ListOpts`.